packages/engine/src/scene/components/VolumetricComponent.ts

```diff
@@ -42,9 +43,12 @@ import { ShadowComponent } from './ShadowComponent'
 import { UVOL1Component } from './UVOL1Component'
 import { UVOL2Component } from './UVOL2Component'
 
-export function handleAutoplay(audioContext: AudioContext, element: HTMLMediaElement) {
+export function handleAutoplay(
+  audioContext: AudioContext,
+  volumetric: State<ComponentType<typeof VolumetricComponent>>
+) {
   const playMedia = () => {
-    element.play()
```

Comment:
element.play() must be called synchronously when a user input event happens, or we break autoplay on iOS

Reply:
Done.